ultravox/training/configs/llama_whisper.yaml

```diff
@@ -0,0 +1,102 @@
+# llama3.1-8b + whisper-medium, for development
+
+exp_name: "llama3.1-8b-whisper"
+text_model: "meta-llama/Meta-Llama-3.1-8B-Instruct"
+audio_model: "openai/whisper-medium"
+
+loss_config:
+  loss_function: "KL_Divergence"
+
+max_steps: 20 # x8x24 = 2,764,800
+
+train_dataset_args:
```

Comment:
Why are datasets being rewritten here? Does the llama_whisper combination require a specific set of train/val/eval datasets that other architectures cannot use?